<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchWidget</class>
 <widget class="QWidget" name="SearchWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>403</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="Russian" country="Russia"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Поиск недвижимости</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QComboBox" name="cityComboBox">
       <item>
        <property name="text">
         <string>Санкт-Петербург</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ленинградская область</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Москва</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Московская область</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="roomsComboBox">
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,1">
     <property name="topMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Цена от</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="fromSpinBox">
       <property name="maximum">
        <number>100000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>до</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="toSpinBox">
       <property name="maximum">
        <number>100000000</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="searchButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial Black</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fromSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SearchWidget</receiver>
   <slot>onSpinBoxValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SearchWidget</receiver>
   <slot>onSpinBoxValueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>229</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>roomsComboBox</sender>
   <signal>highlighted(int)</signal>
   <receiver>SearchWidget</receiver>
   <slot>onRoomsClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>538</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>SearchWidget</receiver>
   <slot>onSearchButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>278</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onRoomsClicked()</slot>
  <slot>onSpinBoxValueChanged()</slot>
  <slot>onSearchButtonClicked()</slot>
 </slots>
</ui>
